integration/modeshape-jbossas-integration-tests/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,10 @@
+<jboss-deployment-structure>
```

Comment:
@hchiorean , This explicit dependency allows to start local ES node before running test